### ISSUE TITLE
offside db postgresql

### ISSUE THREAD

**Issue by geeks121** [davidjameshowell/vaultwarden_heroku#32]
i tried offside db using postgresql from bit.io but nothing happened

**Comment by davidjameshowell** [comment]
Could you provide more details on the issue you are facing and what you have done thus far to help troubleshoot?

**Comment by geeks121** [comment]
i tried with heroku verified : 0
using postgresql from bit.io and the setting above not triggered to use offside DB . as i read the readme ,, to use offside db i should set heroku verified to 0 right? ..

and if i set 
heroku verified : 0
use_psql : 0
the bot will setup without any database 
 
i setup db connection in secrets github ..

**Comment by davidjameshowell** [comment]
If you use heroku_verified as 0 and use_psql as 0, you will essentially be left with no database configuration as we expect use_psql to be true of heroku_verified to be false since the only supported workflow would be that you are supplying your own database URL in that case.

This isn't clear and I will look into failing the workflow if there is not at least use_psql true for heroku_verified false.

**Comment by ncomerci** [comment]
HEROKU_VERIFIED variable is hardcoded in `deploy.yml` and `update.yml`, simple fix [here](https://github.com/ncomerci/vaultwarden_heroku/commit/37c5941ca3fb96c28edb0fdc723e43cc838f3061)

**Comment by davidjameshowell** [comment]
The only real reason it is hard coded is any value that is listed in secret becomes redacted in logs, which subsequently means anything with a "1" becomes an asterisk. Can make debugging a pain.

**Comment by ncomerci** [comment]
Yes, I get your point but I ran into the same problem mentioned here when I did my deploy and I solved it this way so I could change it quickly. I think it would be good to clarify this issue in the readme because I think it is not very clear 🙌🏻😉

**Comment by aditya-shri** [comment]
> Yes, I get your point but I ran into the same problem mentioned here when I did my deploy and I solved it this way so I could change it quickly. I think it would be good to clarify this issue in the readme because I think it is not very clear 🙌🏻wink

Hi bro, I used the bit.io database but after deployment heroku is giving following error:

`2022-04-14 20:09:39.204][panic][ERROR] thread 'main' panicked at 'Failed to disable Foreign Key Checks during migrations: DatabaseError(__Unknown, "QueryParsingError: Unsupported command: SET CONSTRAINTS ALL DEFERRED\nvalidate_unsupported_cmd")': src/db/mod.rs:449`

Any suggestion?
